README.md

```diff
@@ -346,6 +346,13 @@ Specify the option via cmake:
 cmake .. -DUSE_CAFFE2=ON
 ```
 
+#### Build without Caffe
+
+Caffe remains the default backend for DeepDetect though it can be deactivated with cmake. However, at least one library needs to be specified:
+```
+cmake .. -DUSE_CAFFE=NO -DUSE_XGBOOST=ON
```

Comment:
Maybe it should be `-DUSE_CAFFE=OFF` instead of NO, to match other flags, and also avoid typing NO instead of ON or ON instead of NO.

Reply:
Thanks, this is fixed!